app/src/main/AndroidManifest.xml

```diff
@@ -17,6 +17,7 @@
 
     <application
         android:name=".MyApplication"
+        android:largeHeap="true"
```

Comment:
@the-dagger Making the app use more memory and using the RGB 565 to load bitmaps solved the crash I was facing on my device.